Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -23,273 +23,220 @@
 * THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-import CryptoKit
-import Foundation
+#if swift(>=5.9)
 
-@objc(WebCryptoErrorCodes)
-public enum ErrorCodes: Int {
-    case success = 0
-    case wrongTagSize = 1
-    case encryptionFailed = 2
-    case encryptionResultNil = 3
-    case invalidArgument = 4
-    case tooBigArguments = 5
-    case decryptionFailed = 6
-    case hashingFailed = 7
-}
+    #if SWIFTCPP_INTEROP
```

Comment:
Do we actually need this SWIFTCPP_INTEROP check? Would we ever try to compile this code when interop is not available?

Reply:
The swift generator code is triggered for instance on Monterey so I am just guarding the whole file.

Reply:
also I am being explicit about it honestly. `swift(>=5.9)` might just be enough but that is not really informative.